src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java

```diff
@@ -427,9 +414,6 @@ else if(model instanceof AcogReticle) {
                 .setupModernMagazineAnimations("f2000", 
                 		Magazines.M4A1Mag, 
                 		Magazines.M38Mag, 
-                		Magazines.Stanag100,
-                		Magazines.Stanag50,
-                		Magazines.Stanag60,
```

Comment:
Why are they getting removed?

Reply:
For reasons of realism in real life, the F2000 does not support Drum magazine.

Reply:
The maximum compatible would be a 40 or 45 bullet charger but it does not exist